includes/polling/os/gepulsar.inc.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
Could you please move this file to the new format, see https://docs.librenms.org/Developing/os/Initial-Detection/#os-discovery for more info

Reply:
I just got to quit copying code and pay more attention to the documentation! LOL

Reply:
We changed this not too long ago, sorry for the inconvenience

Reply:
Not a problem.  Also updated the snmpre and json test files so Travis will be happy.